libs/feature/editor/src/lib/+state/editor.effects.spec.ts

```diff
@@ -167,6 +172,16 @@ describe('EditorEffects', () => {
     })
   })
 
+  describe('cleanRecordAttachments$', () => {
+    it('dispatch markRecordAsChanged', () => {
+      actions = hot('-a-|', {
+        a: EditorActions.saveRecordSuccess(),
+      })
+      const expected = hot('---|')
+      expect(effects.cleanRecordAttachments$).toBeObservable(expected)
+    })
```

Comment:
I don't get this test... Is it the proper `it` description?
Also, the way I read the marble test, no emit is expected on the observable. Is it really what we want, or am I missing something?

Reply:
Forgot to change the it after the copy pasta. The observable returns void so I don't know if I can marble that differently ?

Reply:
The problem here is that no event is different from an event with no content. Searching on SO, most people expect an undefined tick: https://stackoverflow.com/questions/64258427/how-do-i-marble-test-an-observable-of-void-values